resources/migrations/001_update_migrations.yaml

```diff
@@ -9288,6 +9288,252 @@ databaseChangeLog:
                   name: type
             indexName: idx_collection_type
 
+  - changeSet:
+      id: v51.2024-09-26T03:01:00
+      author: escherize
+      comment: iff enable-embedding is set, copy -> enable-embedding-interactive
+      preConditions:
+        - onFail: MARK_RAN
+        - or:
+            - and:
+                - dbms:
+                    type: postgresql
+                - sqlCheck:
+                    expectedResult: 1
+                    sql: SELECT count(*) FROM setting WHERE key = 'enable-embedding';
```

Comment:
How's this work?

If there is a setting at setting.key `'enable-embedding'`, then:

```
INSERT INTO setting (key, value)
SELECT 'enable-embedding-interactive', value
FROM setting
WHERE key = 'enable-embedding';
```

otherwise do nothing.

Reply:
and we do that for all 5 new settings: 2 for origins, and 3 for enabling embedding.

Reply:
I _think_ this is unnecessary? At least on Postgres the `INSERT INTO ... SELECT` won't insert anything if there's nothing returned from the `SELECT`. But it doesn't hurt, so not going to block on it.